docker/search/Dockerfile.search

```diff
@@ -0,0 +1,10 @@
+FROM ubuntu:16.04
+
+RUN apt-get update && apt-get install -y maven openjdk-8-jdk-headless
+
+RUN mkdir /code
+
+ADD . /code
+
+WORKDIR /code
+RUN mvn install
```

Comment:
sometimes maven won't download specific packages until we run a specific command. I think there is a command that we can run here which will pre-download the packages during docker build so that we don't have to download them every time we start the container. I'll have a look

Reply:
done